File: Tratamento de Exceções em Java e Python/Trabalho/Exemplos Praticos/Ex001.java
public class ExceptionExample {
    public static void main(String[] args) {
        try {
            int[] numbers = {1, 2, 3};
            System.out.println(numbers[3]); // This will throw an ArrayIndexOutOfBoundsException
        } catch (ArrayIndexOutOfBoundsException e) {
            System.out.println("An error occurred: " + e.getMessage());
        } finally {
            System.out.println("The 'try catch' is finished.");
        }
    }
}

File: Tratamento de Exceções em Java e Python/Trabalho/Exemplos Praticos/Ex001.py
def exception_example():
    try:
        numbers = [1, 2, 3]
        print(numbers[3])  # This will raise an IndexError
    except IndexError as e:
        print("An error occurred:", e)
    else:
        print("No errors occurred.")
    finally:
        print("The 'try except' is finished.")

exception_example()
